web/packages/build/package.json

```diff
@@ -54,7 +51,6 @@
     "@types/webpack": "^5.28.5",
     "@vitejs/plugin-react-swc": "^3.7.0",
     "babel-loader": "^8.2.5",
-    "core-js": "^3",
```

Comment:
I don't think a polyfill which is not imported anywhere, has any effect.
And we don't import `core-js`.